demo/components/toggle-demo.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {Toogle} from '../../src';
+
+export interface State {
+    checked: boolean,
+}
+
+export class ToogleDemo extends React.Component<{}, State> {
+    state = {
+        checked: false,
+    }
+    render () {
+        return <table>
+            <tbody>
+                <tr>
+                    <th>
+                        Controled
+                        <Toogle
+                            label='with icons'
+                            checked={this.state.checked}
+                            onChange={(checked: boolean) => this.setState({checked})}
+                        />
+                    </th>
+                    <th>
+                        Controled RTL
+                        <Toogle
+                            label='with icons'
+                            rtl
+                            checked={this.state.checked}
+                            onChange={(checked: boolean) => this.setState({checked})}
+                        />
+                    </th>
+                    <th>
+                        Off
+                        <Toogle/>
+                    </th>
+                    <th>
+                        On
+                        <Toogle checked/>
+                    </th>
+                    <th>
+                        Disabled off
```

Comment:
I would rephrase this as "Disabled and off", otherwise this reads as if `disabled` option is off.